crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -68,11 +69,21 @@ impl ActionHandler for BridgeLock {
             .expect("current source should be set before executing action")
             .source_action_index;
 
+        // map asset to trace prefixed asset for deposit, if it is not already
+        let deposit_asset = match self.asset.as_trace_prefixed() {
+            Some(asset) => asset.clone(),
+            None => state
+                .map_ibc_to_trace_prefixed_asset(&allowed_asset)
+                .await
+                .wrap_err("failed to map IBC asset to trace prefixed asset")?
+                .ok_or_eyre("mapping from IBC prefixed bridge asset to trace prefixed not found")?,
+        };
```

Comment:
```suggestion
        let deposit_asset = if self.asset.is_ibc_prefixed() {
            state
                .map_ibc_to_trace_prefixed_asset(&allowed_asset)
                .await
                .wrap_err("failed to map IBC asset to trace prefixed asset")?
                .ok_or_eyre("mapping from IBC prefixed bridge asset to trace prefixed not found")?
        } else {
            self.asset
        };
```

don't need to close `self.asset`